cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1113,7 +1113,7 @@ public static String clusterCaKeySecretName(String cluster)  {
     }
 
     @SafeVarargs
-    protected static Map<String, String> mergeLabelsOrAnnotations(Map<String, String> internal, Map<String, String>... templates) {
+    public static Map<String, String> mergeLabelsOrAnnotations(Map<String, String> internal, Map<String, String>... templates) {
```

Comment:
I think there is for whatever reason another copy of this method in the `Util` class: https://github.com/strimzi/strimzi-kafka-operator/blob/f599987efc297f238762f0e5bbbb2cb3c407a59e/operator-common/src/main/java/io/strimzi/operator/common/Util.java#L300 

Probably better to use that one than change the visibility of this one. (unless this one works differently and you need to use it)